osgp/shared/osgp-ws-core/src/main/resources/schemas/notification-ws-core.xsd

```diff
@@ -40,8 +40,9 @@
 
   <xsd:simpleType name="NotificationType">
     <xsd:restriction base="xsd:string">
-      <xsd:enumeration value="DEVICE_AUTHORIZATION_UPDATED" />
       <xsd:enumeration value="DEVICE_UPDATED" />
+      <xsd:enumeration value="SET_LIGHT_SCHEDULE" />
```

Comment:
Consider not using public lighting or tariffswitching specific values in ws-core

Reply:
All notifications of ws-admin, ws-core, ws-publiclighting and ws-tariffswitching are sent to the receiving end-point by ws-core.

Reply:
https://github.com/OSGP/open-smart-grid-platform/pull/27/commits/c77ef04ca769d51c4c1c8a4f994773e60f7f4ee3

Reply:
For ws-publiclighting and ws-tariffswitching the notifications have been implemented in both components.